cicecore/shared/ice_calendar.F90

```diff
@@ -126,16 +126,18 @@ module ice_calendar
          force_restart_now, & ! force a restart now
          write_history(max_nstrm) ! write history now
 
-      character (len=1), public :: &
+      character (len=2), public :: &
          npt_unit,            & ! run length unit, 'y', 'm', 'd', 'h', 's', '1'
          npt0_unit,           & ! original run length unit, 'y', 'm', 'd', 'h', 's', '1'
          histfreq(max_nstrm), & ! history output frequency, 'y','m','d','h','1'
-         dumpfreq               ! restart frequency, 'y','m','d'
+         dumpfreq(max_nstrm)    ! restart frequency, 'y','m','d'
```

Comment:
Here we could take the opportunity to adjust the comment to also mention `h` and `1`.